components/CommentInput.tsx

```diff
@@ -7,12 +7,13 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import Avatar from "./Avatar";
 
 function CommentInput() {
+  const [toast, setToast] = useState(false);
```

Comment:
```suggestion
  const [showToast, setShowToast] = useState(false);
```
이름이 무슨 기능을 하는지 확실하면 더 좋을거 같아요!

Reply:
오 맞네요! 감사합니다. 수정하겠습니다.